index.html

```diff
@@ -0,0 +1,670 @@
+    />
+    <title>Youtube Page</title>
+  </head>
+  <main>
+  <body>
+    <nav class="flex">
+      <div id="left-navbar" class="flex">
+        <a href="#"><img src="assets/hamburger.png" alt="" /></a>
+        <a href="#"><img src="assets/youtubelogos.png" alt="" /></a>
+      </div>
+      <div id="mid-navbar" class="flex">
+        <input id="search-bar" type="text" placeholder="Search" />
+        <button id="search-button">
+          <i class="fa fa-search" aria-hidden="true"></i>
+        </button>
+      </div>
+
+      <div id="right-navbar" class="flex">
+        <a href="#"
+          ><img src="assets/nav1.png" alt=""
+        /></a>
+        <a href="#"><img src="assets/nav2.png" alt="" /></a>
+        <a href="#"><img src="assets/nav4.png"
+            alt=""
+        /></a>
+        <a href="#"><img src="assets/nav3.png" 
+            alt=""
+        /></a>
+        
+      </div>
+    </nav>
+
+    <div class="d-flex space-between">
+      <div class="main-fc d-flex flex-column">
+        <div class="container">
+          <div class="main-video">
+            <iframe
+              width="853"
+              height="480"
+              src="https://www.youtube.com/embed/YbYWhdLO43Q"
+              frameborder="0"
+              allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
+            ></iframe>
+            <div class="video-title">
+              <p>
+                This Unicorn Changed the Way I Poop -
+                <a
+                  href="https://www.youtube.com/results?search_query=%23SquattyPotty"
+                  >#SquattyPotty</a
+                >
+              </p>
+            </div>
+            <div id="video-below" class="flex flex-horizontal">
+              <div class="video-views">
+                <p class="smaller-gray">38,063,145 views • Oct 6 2015</p>
+              </div>
+              <div class="flex">
+                <i class="fa fa-thumbs-up icon-spacing" aria-hidden="true"
+                  > 115K</i
+                >
+                <i class="fa fa-thumbs-down icon-spacing" aria-hidden="true"
+                  > 5.8K</i
+                >
+                <i class="fa fa-share icon-spacing"> SHARE</i>
+                <i class="fa fa-save icon-spacing" aria-hidden="true"> SAVE</i>
+                <i class="fa fa-ellipsis-h icon-spacing" aria-hidden="true"></i>
+              </div>
+            </div>
+            <hr />
+            <br />
+          </div>
+
+          <article class="video-summary">
+            <div class="page-profile">
+              <div class="user-description">
+                <div class="page-name"></div>
+                <div class="align-center">
+                  <img
+                    class="images inline"
+                    src="assets/user-profile.jpg"
+                    alt="Page Profile"
+                    width="40"
+                    height="40"
+                  />
```

Comment:
please remove inline styling. The image tag should only contain the alt element as it is used for screen readers.